admin/sql/updates/2019-01-04-add-user-login-id.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+-- Add column for alternative user id for login purpose to user table
+ALTER TABLE "user" ADD COLUMN user_login_id VARCHAR;
```

Comment:
we will need this script to fill out the column for existing users also, because each user must have a login ID, no matter what.